### ISSUE TITLE
Error: stream.push() after EOF

### ISSUE THREAD

**Issue by pfrazee** [mafintosh/hypercore-protocol#14]
This is occurring frequently in replication

```
dat error (7552ms) id=1 chan=bf2212..8d type=utp host=72.182.35.118 key=a60bdd..18 Error: stream.push() after EOF
    at readableAddChunk (/Users/paulfrazee/my/hypercore/node_modules/readable-stream/lib/_stream_readable.js:193:15)
    at Protocol.Readable.push (/Users/paulfrazee/my/hypercore/node_modules/readable-stream/lib/_stream_readable.js:171:10)
    at Protocol.ping (/Users/paulfrazee/my/hypercore/node_modules/hypercore-protocol/index.js:188:15)
    at Protocol._kick (/Users/paulfrazee/my/hypercore/node_modules/hypercore-protocol/index.js:177:10)
    at Timeout.kick [as _onTimeout] (/Users/paulfrazee/my/hypercore/node_modules/hypercore-protocol/index.js:73:10)
    at ontimeout (timers.js:365:14)
    at Timer.unrefdHandle (timers.js:471:5)
```

**Comment by pfrazee** [comment]
Solved in 6.3.1

**Comment by bnewbold** [comment]
Can this be closed?
